tests/foreman/cli/test_product.py

```diff
@@ -318,23 +320,55 @@ def test_positive_assign_http_proxy_to_products(module_org):
         assert r['http-proxy']['http-proxy-policy'] == 'none'
 
 
-@pytest.mark.stubbed
 @pytest.mark.tier2
-def test_positive_product_sync_state():
+@pytest.mark.skip_if_open('BZ:1999541')
+def test_positive_product_sync_state(module_org):
     """hammer product info shows correct sync state.
 
     :id: 58af6239-85d7-4b8b-bd2d-ab4cd4f29840
 
-    :BZ: 1803207
+    :BZ: 1803207,1999541
 
     :customerscenario: true
 
+    :CaseLevel: Component
```

Comment:
These aren't strictly necessary, this metadata is specified in the module level docblock (L7 and L15), which will apply to all classes and functions within the module.

Reply:
Okay will update it.